### ISSUE TITLE
Support `diff` option in `runme fmt`

### ISSUE THREAD

**Issue by mxsdev** [stateful/runme#312]
Would be nice to have the output of `runme fmt` as a diff/patch file, especially with project support.

**Comment by sourishkrout** [comment]
like e.g. `runme fmt --patch` or as default output @mxsdev?

**Comment by mxsdev** [comment]
@sourishkrout 

Yeah, I'm imagining it as a flag between patch and "normal" outout.

I personally also think it makes more sense as a default output format, since it is more useful for e.g. reviewing changes before actually making them.
